src/app/routes/dashboard/dashboard.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component } from '@angular/core';
+import { Router, Routes, ActivatedRoute } from '@angular/router';
+
+import { RequestService } from "../../RequestService/requests";
+
+@Component({
+  selector: 'app-dashboard',
+  templateUrl: './dashboard.component.html',
+  styleUrls: ['./dashboard.component.css']
+})
+export class DashboardComponent {
+  featureds: any = [];
+  events: any = [];
+  categories: any = [];
+  departments: any = [];
+
+  constructor(private requestService: RequestService, private route: ActivatedRoute, private router: Router) {
+    // get featureds data
+    this.requestService.get('/pages/featureds', (data) => {
```

Comment:
Later we might look into generalizing these. But for now, there are many, more important things we need to accomplish.